CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ option(WITH_FORTRAN     "Whether to build Fortran bindings"                 NO)
 option(WITH_HDF5        "Whether to build with HDF5 support"               YES)
 option(WITH_REDLAND     "Whether to build with RDF support (using librdf)" YES)
 option(WITH_THREADS     "Whether to build with threading support"           NO)
-option(WITH_DOC         "Whether to build documentation using doxygen"     YES)
+option(WITH_DOC         "Whether to build documentation using doxygen"      NO)
```

Comment:
Is this still only explicitly to build with doxygen, or does it now include the Sphinx build? If that latter is true, I would update the comment here, changing `doxygen` to `Sphinx`.

Reply:
This include both the doxygen and Sphinx documentation. Changed it to sphinx+doxygen